diff --git a/ros/src/foxglove_bridge/src/message_definition_cache.cpp b/ros/src/foxglove_bridge/src/message_definition_cache.cpp
--- a/ros/src/foxglove_bridge/src/message_definition_cache.cpp
+++ b/ros/src/foxglove_bridge/src/message_definition_cache.cpp
@@ -10,12 +10,7 @@
 #include <unordered_set>
 #include <utility>
 
-#include <ament_index_cpp/version.h>
-#if AMENT_INDEX_CPP_VERSION_GTE(1, 13, 2)
-#include <ament_index_cpp/get_package_share_path.hpp>
-#else
 #include <ament_index_cpp/get_package_share_directory.hpp>
-#endif
 #include <ament_index_cpp/get_resource.hpp>
 #include <rcutils/logging_macros.h>
 
@@ -223,26 +218,13 @@ MessageSpec MessageDefinitionCache::load_message_spec(
 
   // Get the package share directory, or throw a PackageNotFoundError
   std::filesystem::path share_dir;
-#if AMENT_INDEX_CPP_VERSION_GTE(1, 13, 2)
-  share_dir = ament_index_cpp::get_package_share_path(package);
-#else
   share_dir = ament_index_cpp::get_package_share_directory(package);
-#endif
 
   // Get the rosidl_interfaces index contents for this package
   std::string index_contents;
-#if AMENT_INDEX_CPP_VERSION_GTE(1, 13, 0)
-  ament_index_cpp::PathWithResource path_with_resource =
-    ament_index_cpp::get_resource("rosidl_interfaces", package);
-  if (path_with_resource.resourcePath == std::nullopt) {
-    throw DefinitionNotFoundError(definition_identifier.package_resource_name);
-  }
-  index_contents = path_with_resource.contents;
-#else
   if (!ament_index_cpp::get_resource("rosidl_interfaces", package, index_contents)) {
     throw DefinitionNotFoundError(definition_identifier.package_resource_name);
   }
-#endif
 
   // Find the first line that ends with the filename we're looking for
   const auto lines = split_string(index_contents);
