diff --git a/src/ros_utils/message_definition_cache.cpp b/src/ros_utils/message_definition_cache.cpp
--- a/src/ros_utils/message_definition_cache.cpp
+++ b/src/ros_utils/message_definition_cache.cpp
@@ -14,12 +14,7 @@
 
 #include "message_definition_cache.hpp"
 
-#include <ament_index_cpp/version.h>
-#if AMENT_INDEX_CPP_VERSION_GTE(1, 13, 2)
-#include <ament_index_cpp/get_package_share_path.hpp>
-#else
 #include <ament_index_cpp/get_package_share_directory.hpp>
-#endif
 #include <ament_index_cpp/get_resource.hpp>
 #include <ament_index_cpp/get_resources.hpp>
 #include <fstream>
@@ -72,11 +67,7 @@ const MessageSpec& MessageDefinitionCache::load_message_spec(const std::string& d
   }
   std::string package = match[1];
   std::string share_dir;
-#if AMENT_INDEX_CPP_VERSION_GTE(1, 13, 2)
-  share_dir = ament_index_cpp::get_package_share_path(package);
-#else
   share_dir = ament_index_cpp::get_package_share_directory(package);
-#endif
   std::ifstream file{share_dir + "/msg/" + match[2].str() + ".msg"};
 
   std::string contents{std::istreambuf_iterator(file), {}};
